Makefile

```diff
@@ -10,7 +11,8 @@ MIN_KUBERNETES_VERSION ?= 1.25.0
 MIN_OPENSHIFT_VERSION ?= 4.12
 
 TEMPO_IMAGE ?= docker.io/grafana/tempo:$(TEMPO_VERSION)
-TEMPO_QUERY_IMAGE ?= docker.io/grafana/tempo-query:$(TEMPO_QUERY_VERSION)
+JAEGER_QUERY_IMAGE ?= docker.io/jaegertracing/jaeger-query:$(JAEGER_QUERY_VERSION)
+TEMPO_QUERY_IMAGE ?= ghcr.io/frzifus/tempo-query:$(TEMPO_QUERY_VERSION)
```

Comment:
FTR; this image is a custom build of this PR.
- https://github.com/grafana/tempo/pull/3840